website/content/en/docs/migration/v0.18.0.md

```diff
@@ -3,117 +3,4 @@ title: v0.18.0
 weight: 999982000
 ---
 
-## (Optional) Add livenessProbe check for Ansible-based operators
```

Comment:
Are not these the 0.18.0 steps. Why are removing them? 
Shows that the 0.18.1 was not generated maybe because has no migration steps.

Reply:
Maybe I did it wrong.

Reply:
Did you run `GEN_CHANGELOG_TAG=v0.18.1 make gen-changelog`?

Reply:
Of course I did says my brain. But the CLI history does not lie. Apparently I did not. `GEN_CHANGELOG_TAG=v0.18.0 make gen-changelog`.

Reply:
These are the migrations from v0.18.0 to v0.18.1